dvc/tree/base.py

```diff
@@ -75,8 +75,8 @@ def __init__(self, repo, config):
     @cached_property
     def hash_jobs(self):
         return (
-            self.config.get("hash_jobs")
-            or (self.repo and self.repo.config["core"].get("hash_jobs"))
+            self.config.get("checksum_jobs")
```

Comment:
This is a nasty one caught by the verification. Config schema never allowed for `hash_jobs`, so we were effectively not using `checksum_jobs` at all all this time. But now we do.